Serverp/src/Server.java

```diff
@@ -85,21 +106,92 @@ private static void SetUpConnections()
 	}
 
 	/*
-	 * Runnable class to handle instances of clients that are connected
+	 * Writes a String to a client through socket socket
+	 */
+	private static void writeToCLient(String string, Socket socket) throws IOException
+	{
+
+		pw = new PrintWriter(socket.getOutputStream(), true);
+		pw.println(string);
+
+	}
+
+	
+
+	/*
+	 * Writes an ArrayList of objects to the client through socket
+	 */
+	private static void writeToClient(ArrayList<Object> list, Socket socket) throws IOException
```

Comment:
Data Manager will return a List<Object> rather than ArrayList<Object> (in case we decide that another form of list would be more efficient in the future), so the first parameter should be changed accordingly.